spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1631,17 +1639,15 @@ marshaller, and a `rootTagName`. A Java object is passed to a marshaller (typica
 standard Spring OXM Marshaller) which writes to a `Resource` by using a custom event
 writer that filters the `StartDocument` and `EndDocument` events produced for each
 fragment by the OXM tools. The following example uses the
-`MarshallingEventWriterSerializer`:
```

Comment:
I cannot find `MarshallingEventWriterSerializer` anywhere in the code base. Is this obsolete? I used the `StaxEventItemWriter` in the description to be consistent with the xml/java code examples.

Reply:
done